src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx

```diff
@@ -63,6 +63,13 @@ export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
         onBlur={handleBlur}
       />
 
+      {/* render email error separately to have its design aligned with PhoneNumberInput */}
```

Comment:
Have you tried to use input's `error` prop? if PhoneNumberInput error and Input error different let's update PhoneNumberInput error.
`error={touched.email && errors.email}`

Reply:
Yes, I've tried. if we use the error prop and want to have email and phone number input styles to be the same, we would need to:

- update the styles of Select component (for red border) in PhoneNumberInput
- add logic to dynamically update those styles when the Input next to Select is hovered on 

which is an overkill, in my opinion. but I'm fine with your suggestion if you think going through styles updates makes sense.

Reply:
ok I'll give it a try

Reply:
this works! thanks